rfcs/0028-change-since-relative-timestamp/0028-change-since-relative-timestamp.md

```diff
@@ -1 +1,76 @@
-In review, see <https://github.com/nervosnetwork/rfcs/pull/221>
+---
+Number: "0028"
+Category: Consensus (Soft Fork)
```

Comment:
Category must be in either `Standards Track` or `Informational`. Otherwise a new category with its description should be added first.

I prefer to move forward with existing category first. After new RFCs merged (will be in draft status) we can come back to the category issue and see if we need to update RFC categories.